src/main.ts

```diff
@@ -1,8 +1,8 @@
-import * as correlation from "./correlation";
+import * as correlation from "./correlation.js";
```

Comment:
Do these imports work as e.g.?

```js
import * as correlation from "./correlation";
```

Because importing `.js` into a `.ts` file shouldn't need to happen.

Reply:
They don't :(. I had to change it to make it work.

Reply:
https://www.totaltypescript.com/relative-import-paths-need-explicit-file-extensions-in-ecmascript-imports I think we have to use .js in our imports, which seems to be pretty standard across the ecosystem: https://github.com/sindresorhus/got/blob/main/source/core/errors.ts#L3